### PR TITLE
new links

### PR DESCRIPTION
**Pull request by JoshMacSween**
Hey team! Just a little weekday PR here, just fixing some of the dead links, let me know if you want them in the nav bar or in the footer.

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -10,4 +10,16 @@ def meetup
...
+  end
+
+  def slack
+  end
+
+  def gs
+  end
+
+  def learn_ror
+  end
```
Comment (alanvardy) [inline app/controllers/pages_controller.rb]: It seems like we have 2 pages for learning resources.. lets combine them into one and just call it resources

File: app/views/pages/slack.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<h1>Slack</h1>
+
+<p>you can connect with all us here!</p>
+
+<a href="https://devedmonton-invite.herokuapp.com/">Slack</a>
```
Comment (alanvardy) [inline app/views/pages/slack.html.erb]: You can grab the dev edmonton image from the current YEGRB page if you like

File: app/views/pages/opportunities.html.erb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<h1>Opportunities</h1>
+
+<p>Wowee! Lots of jobs here</p>
```
Comment (alanvardy) [inline app/views/pages/opportunities.html.erb]: Good placeholder page. I plan on replacing this with a scaffold down the road (so logged in Users can add opportunities)

File: app/views/layouts/_navbar.html.erb [hunk 4]
```diff
@@ -17,6 +17,18 @@
...
+      </li>
+      <li>
+      <%= link_to "Getting Started", gs_path, class: 'nav-link'%>
+      </li>
+      <li>
+      <%= link_to "Learn Ruby on Rails", learn_ror_path, class: 'nav-link' %>
+      </li>
     </ul>
     <ul class="navbar-nav">
       <li class="nav-item">
```
Comment (alanvardy) [inline app/views/layouts/_navbar.html.erb]: Can you fix the indentation?

It should look like:
```html
<ul>
      <li>
          <%= link_to "Getting Started", gs_path, class: 'nav-link'%>
      </li>
      <li>
          <%= link_to "Learn Ruby on Rails", learn_ror_path, class: 'nav-link' %>
      </li>
</ul>
```

4 spaces for html, not two (get me to help you setting VS Code to recognise erb and HTML files)

**Comment by alanvardy** [comment]
Could you add tests for your new pages?

Just follow the code that I have already written and run them using `rails t`

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (JoshMacSween) [reply app/controllers/pages_controller.rb]: Sure thing! I’ll do that when I get in. 
What would you like in the navbar and what do you want in the footer?

**Comment by JoshMacSween** [comment]
Sure to all the things! :) I’ll fix them up when I get in

**Comment by alanvardy** [comment]
Oh, and please reference the issue when you post a pull request :) you can say `closes #1`

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (alanvardy) [reply app/controllers/pages_controller.rb]: Could you go read a few blog posts about this and come back to me with an answer?

I feel like I want the main sections to be in the navbar, and the footer should be reserved for contact admin, join slack, find on GitHub etc. (Things that take a person away from the website).

This is probably something where after doing some research you can open up an issue describing what work needs to be done.

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -15,6 +15,9 @@
     <div class="container">
       <%= yield %>
     </div>
+    #Stolen Sytax
+    <%= image_tag %>
```
Comment (alanvardy) [inline app/views/layouts/application.html.erb]: This needs arguments, i.e. `<%= image_tag "picture.jpg" %>`

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -15,6 +15,9 @@
     <div class="container">
       <%= yield %>
     </div>
+    #Stolen Sytax
```
Comment (alanvardy) [inline app/views/layouts/application.html.erb]: This is not a valid comment in an erb file

Comment (alanvardy) [reply app/views/layouts/application.html.erb]: You need to use either <% #stolen syntax %> or an html comment

**Pull request merged**